File: week_3/1.c
#include <stdio.h>
int main() {
    int q,w,e,z,x,c;
    printf("Enter 1 number: ");
    scanf("%d", &q);
    printf("Enter 2 number: ");
    scanf("%d", &w);
    printf("Enter 3 number: ");
    scanf("%d", &e);
    z = q+e;
    x = q+w;
    c = w+e;
    if (z>x && z>c) printf("%d + %d = %d", q, e, z);
    else if (x>z && x>c) printf("%d + %d = %d", q, w, x);
    else printf("%d + %d = %d", w, e, c);
}

File: DATA STRUCTURES/9/1.py
def insertionSort(arr):
    for n, i in enumerate(arr[1:], 1):
        for nk, ik in enumerate(arr[:n]):
            if ik > i: 
                arr.insert(nk, arr.pop(n))
                break
        print('Round', n, arr)            

 
inp = list(map(int, input("Enter list for number: ").split(",")))
print("Before sort:", inp)
insertionSort(inp)
print("After sort:", inp)